x/ccv/provider/keeper/proposal.go

```diff
@@ -172,38 +165,36 @@ func (k Keeper) StopConsumerChain(ctx sdk.Context, chainID string, lockUbd, clos
 	k.ConsumeSlashAcks(ctx, chainID)
 	k.DeletePendingPackets(ctx, chainID)
 
-	// release unbonding operations if they aren't locked
+	// release unbonding operations
 	var vscIDs []uint64
-	if !lockUbd {
-		// iterate over the consumer chain's unbonding operation VSC ids
-		k.IterateOverUnbondingOpIndex(ctx, chainID, func(vscID uint64, ids []uint64) (stop bool) {
-			// iterate over the unbonding operations for the current VSC ID
-			var maturedIds []uint64
-			for _, id := range ids {
-				unbondingOp, found := k.GetUnbondingOp(ctx, id)
-				if !found {
-					err = fmt.Errorf("could not find UnbondingOp according to index - id: %d", id)
-					return true // stop the iteration
-				}
-				// remove consumer chain ID from unbonding op record
-				unbondingOp.UnbondingConsumerChains, _ = removeStringFromSlice(unbondingOp.UnbondingConsumerChains, chainID)
-
-				// If unbonding op is completely unbonded from all relevant consumer chains
-				if len(unbondingOp.UnbondingConsumerChains) == 0 {
-					// Store id of matured unbonding op for later completion of unbonding in staking module
-					maturedIds = append(maturedIds, unbondingOp.Id)
-					// Delete unbonding op
-					k.DeleteUnbondingOp(ctx, unbondingOp.Id)
-				} else {
-					k.SetUnbondingOp(ctx, unbondingOp)
-				}
+	// iterate over the consumer chain's unbonding operation VSC ids
+	k.IterateOverUnbondingOpIndex(ctx, chainID, func(vscID uint64, ids []uint64) (stop bool) {
```

Comment:
This just looks weird. The only thing that happened was removing `if !lockUbd` and removing the indentation